src/main/java/by/exadel/internship/entity/Form.java

```diff
@@ -19,6 +21,7 @@
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
+@EqualsAndHashCode(exclude = {"interview", "timeForCallList"})
```

Comment:
here the same 
https://github.com/viktarkolbik/backend_team_four/pull/82/files#r640473592

Reply:
done